packages/dd-trace/src/appsec/iast/context/context-plugin.js

```diff
@@ -0,0 +1,110 @@
+'use strict'
+
+const { storage } = require('../../../../../datadog-core')
+const iastContextFunctions = require('../iast-context')
+const overheadController = require('../overhead-controller')
+const { IastPlugin } = require('../iast-plugin')
+const { IAST_ENABLED_TAG_KEY } = require('../tags')
+const { createTransaction, removeTransaction } = require('../taint-tracking/operations')
+const vulnerabilityReporter = require('../vulnerability-reporter')
+const { TagKey } = require('../telemetry/iast-metric')
+
+class IastContextPlugin extends IastPlugin {
+  startCtxOn (channelName, tag, handler) {
```

Comment:
`tag` can be a single tag or an array of tags but I admit it can be a bit confusing.
It's related with the metrics and I plan to refactor them in another PR